packages/colorpickers/src/styled/ColorPicker/StyledAlpha.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { Range } from '@zendeskgarden/react-forms';
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { colorStyles, thumbStyles, trackStyles, trackLowerStyles } from './StyledHue';
```

Comment:
Should shared styling be abstracted into a `StyledRange` base class instead? Even with this, there seems to be additional crossover that would be cleaner and more maintainable to abstract and subsequently wrapped by both `StyledAlpha` and `StyledHue`.

Reply:
Refactored shared styling to `/common/StyledRange` 👍🏽